app/views/organisations/_separate_website.html.erb

```diff
@@ -1,25 +1,26 @@
-<%= render "govuk_publishing_components/components/heading", {
-    text: organisation.title,
-    heading_level: 2
-} %>
+<% content_for :breadcrumbs do %>
+  <div class="organisations__margin-bottom">
+    <h1 class="visually-hidden"><%= @header.org.title %></h1>
+  </div>
+<% end %>
 
-<div class="separate-website-label">
-  <%= "#{organisation.title} has a" %> <%= link_to "separate website", "#" %>
+<div class="grid-row">
+  <div class="column-one-third organisations__margin-bottom">
+    <%= render "govuk_publishing_components/components/organisation_logo", {
+      organisation: @header.organisation_logo
+    } %>
+  </div>
 </div>
 
-<div class="organisation-body">
-  <%= organisation.body %>
-</div>
+<%= render "govuk_publishing_components/components/notice", {
+  title: "#{organisation.title} has a <a href='/'>separate website</a>".html_safe
+} %>
 
-<div class="parent-organisations">
-  <% if organisation.ordered_parent_organisations.presence %>
-    <%= "#{organisation.title} works with the" %>
-    <%= link_to organisation.ordered_parent_organisations[0]["title"],
-                organisation.ordered_parent_organisations[0]["base_path"] %>
-  <% end %>
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render "govuk_publishing_components/components/govspeak", {
+    } do %>
+      <p><%= organisation.body.html_safe %></p>
```

Comment:
The live site also contains a link to the organisation website at the bottom of this section:

<img width="634" alt="screen shot 2018-06-18 at 13 49 15" src="https://user-images.githubusercontent.com/29889908/41536793-7996ed56-72fe-11e8-8047-f717bcbee787.png">